src/lib/ui/srv_status_component.rb

```diff
@@ -0,0 +1,152 @@
+require "yast"
+Yast.import "Service"
+Yast.import "UI"
+
+module UI
+  class SrvStatusComponent
+    include Yast::UIShortcuts
+    include Yast::I18n
+    include Yast::Logger
+
+    def initialize(service_name, reload: true)
+      @service_name = service_name
+      @reload = reload
+
+      @enabled = service_enabled?
+      @id_prefix = "_srv_status_#{@service_name}"
+    end
+
+    def widget
+      VBox(
+        ReplacePoint(Id("#{id_prefix}_status"), status_widget),
+        VSpacing(),
+        on_boot_widget,
+        VSpacing(),
+        reload_widget
+      )
+    end
+
+    def handle_input(input)
+      case input
+        when "#{id_prefix}_stop"
+          stop_service
+          refresh_widget
+        when "#{id_prefix}_start"
+          start_service
+          refresh_widget
+        when "#{id_prefix}_reload"
+          @reload = Yast::UI.QueryWidget(Id(input), :Value)
+        when "#{id_prefix}_enabled"
+          @enabled = Yast::UI.QueryWidget(Id(input), :Value)
+      end
+    end
+
+    def init_widget
```

Comment:
yes, it is possible. It is attribute, so you just pass it as option, if I remember correctly. I hope something like https://github.com/yast/yast-ycp-ui-bindings/blob/master/examples/Enabling1.rb#L9